test/services/bundle.service.test.js

```diff
@@ -58,9 +79,20 @@ describe('Test transaction bundle upload', () => {
       });
   });
 
-  afterAll(cleanUpTest);
+  test('error thrown if method  type is not PUT or POST', async () => {
+    await supertest(server.app)
+      .post('/4_0_1/')
+      .send(INVALID_METHOD_REQ)
+      .set('Accept', 'application/json+fhir')
+      .set('content-type', 'application/json+fhir')
+      .set('x-provenance', JSON.stringify(SINGLE_AGENT_PROVENANCE))
+      .expect(200)
+      .then(async response => {
+        expect(response.body.resourceType).toEqual('Bundle');
+        expect(response.body.entry[0].response.status).toEqual('400 BadRequest');
```

Comment:
Because there are a bunch of possible 400 errors here, I think we should also check that response.body.entry[0].response.outcome.issue[0].details.text is equal to the expected error message. Something like "Expected requests of type PUT or POST, received GET for test-measure" in this case